modules/apps/29-fee/keeper/grpc_query_test.go

```diff
@@ -24,7 +24,7 @@ func (suite *KeeperTestSuite) TestQueryIncentivizedPackets() {
 	testCases := []struct {
 		name     string
 		malleate func()
-		expPass  bool
+		errMsg   string
```

Comment:
This file relies entirely on error messages as all the errors we get from GRPC queries are not wrapped